src/main/java/dev/nolij/zson/Zson.java

```diff
@@ -232,9 +233,6 @@ public static <T> T map2Obj(@NotNull Map<String, ZsonValue> map, @NotNull Class<
 			T object = type.getDeclaredConstructor().newInstance();
 			for (Field field : type.getDeclaredFields()) {
 				if(!shouldInclude(field, false)) continue;
-				if (!map.containsKey(field.getName())) {
-					throw new IllegalArgumentException("Missing field " + field.getName() + " in map");
-				}
 				setField(field, object, map.get(field.getName()).value);
```

Comment:
Won't this throw a NullPointerException when the key doesn't exist?

Reply:
Yes.